src/main/java/com/epam/aidial/core/log/GfLogStore.java

```diff
@@ -297,34 +271,77 @@ static String assembleStreamingResponse(Buffer response) {
             result.set("model", model);
 
             if (usage != null) {
+                MergeChunks.removeIndices(usage);
                 result.set("usage", usage);
             }
             if (statistics != null) {
+                MergeChunks.removeIndices(statistics);
                 result.set("statistics", statistics);
             }
             if (systemFingerprint != null) {
                 result.set("system_fingerprint", systemFingerprint);
             }
 
-            if (content.isEmpty()) {
+            if (choices == null) {
```

Comment:
let's make this code same as above
if (choices != null) {
remove indices and set
}

Reply:
fixed